decoder/incident.go

```diff
@@ -194,6 +195,7 @@ func createIncidentWebhooks(ctx context.Context, db db.DbDetails, oldIncident *I
 			}
 		}
 		areas := MatchStatsGeofence(stop.Lat, stop.Lon)
+		external.UpdateIncidentCount(areas)
```

Comment:
I am not sure that only new incidents are sent here, but there could be changes - the description says total number of incidents.
Should the description be total number of incident changes?
I'm not sure sending these in the webhook method rather than the save method is the right place - is your intention to tie these to webhooks sent or updates of the record?

Reply:
You are right, have to rework to 
```go
if oldIncident == nil {
			areas := MatchStatsGeofence(stop.Lat, stop.Lon)
			external.UpdateIncidentCount(areas)
		}
```
My intention was to reuse `stop, _ := GetPokestopRecord(ctx, db, incident.PokestopId)` which is pulled inside `createIncidentWebhooks`.
Unless we can move that above?

Reply:
Updated desc to match `Total number of incidents updates`.
As those make sense with this check
`oldIncident == nil || (oldIncident.ExpirationTime != incident.ExpirationTime || oldIncident.Character != incident.Character)`